rocAL/rocAL_pybind/examples/rocAL_api_external_source_example.py

```diff
@@ -0,0 +1,80 @@
+import types
+import collections
+import numpy as np
+from random import shuffle
+from amd.rocal.pipeline import Pipeline
+from amd.rocal.plugin.pytorch import ROCALClassificationIterator
+import amd.rocal.fn as fn
+import amd.rocal.types as types
+
+def main():
+
+    batch_size = 3
+    data_dir = "/media/MIVisionX-data/rocal_data/coco/coco_10_img/train_10images_2017/" # Pass the data directory
+    device = "cpu"
+
+    def draw_patches(img, idx, device):
+    #image is expected as a tensor, bboxes as numpy
```

Comment:
bboxes are not used here, please change and move the comment before function definition

Reply:
@swetha097 Please check this

Reply:
Done